Makefile

```diff
@@ -1487,6 +1491,20 @@ manila_kuttl: kuttl_common_prep ceph manila manila_deploy_prep ## runs kuttl tes
 	make kuttl_common_cleanup
 	make cleanup
 
+.PHONY: swift_kuttl_run
+swift_kuttl_run: ## runs kuttl tests for the swift operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${SWIFT_KUTTL_CONF} ${SWIFT_KUTTL_DIR}
```

Comment:
I think it would be a good idea to add here `--namespace ${NAMESPACE}` so that the tests can be run in a different namespace easily

Reply:
I agree.  Since we already have `export NAMESPACE = ${SWIFT_KUTTL_NAMESPACE}` on L1499, it seems like we're just missing `--namespace ${NAMESPACE}` here.

Reply:
done, thank you both